src/govuk/components/summary-list/template.njk

```diff
@@ -7,38 +7,76 @@
   </a>
 {% endmacro -%}
 
+{%- macro _summaryCard(params) %}
+  {%- set headingLevel = params.title.headingLevel if params.title.headingLevel else 2 -%}
+
+  <div class="govuk-summary-card {%- if params.classes %} {{ params.classes }}{% endif %}"{% for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
+    <div class="govuk-summary-card__title-wrapper">
+      {%- if params.title -%}
+        <h{{ headingLevel }} class="govuk-summary-card__title {%- if params.title.classes %} {{ params.title.classes }} {%- endif %}">
+          {{- params.title.html | safe if params.title.html else params.title.text -}}
+        </h{{ headingLevel }}>
+      {%- endif -%}
+
+      {%- if params.actions.items.length -%}
+        <ul class="govuk-summary-card__actions {%- if params.actions.classes %} {{ params.actions.classes }} {%- endif %}">
+          {%- for action in params.actions.items -%}
```

Comment:
Real nitpicky thing, but action links in the 'content' part of the card (≈ line 57) have some funky logic where it only renders within a list element if there is more than one link present, otherwise the link is rendered standalone.

For consistency, would we want to reproduce that logic here too?